tests/by-util/test_chown.rs

```diff
@@ -730,15 +730,15 @@ fn test_chown_file_notexisting() {
     let user_name = String::from(result.stdout_str().trim());
     assert!(!user_name.is_empty());
 
-    let _result = scene
+    let result = scene
```

Comment:
This change causes an "unused variable: result" warning. So I would either keep the underscore or remove `let result =`.